server/index.js

```diff
@@ -48,8 +48,8 @@ app.put("/todo/:id", (request, response) => {
 
 app.delete("/todo/:id", (request, response) => {
   var id = parseInt(request.params.id);
-  if (todos.filter((todo) => todo.id == id).length !== 0) {
-    todos = todos.filter((todo) => todo.id !== id);
+  if (todos.some((todo) => todo.id === id).length !== 0) {
+    todos = todos.some((todo) => todo.id !== id);
```

Comment:
This will break the deletion. I mentioned to change only the "filter" in the IF condition not inside it